### DISCUSSION TITLE
.NET Conf 2022

### DISCUSSION THREAD

**Discussion by sbwalker** [oqtane/oqtane.framework#2489] [General]
I will be presenting a session at .NET Conf 2022 tomorrow titled "Extending apps to mobile and desktop with .NET MAUI and Oqtane" which is part of the OSS Spotlight section of the virtual conference. 

Nov 8, 2022
20:00 - 20:30 PST

In this session we are going to showcase the exciting Blazor Hybrid support which is included in .NET MAUI. Specifically we will focus on the BlazorWebView control that allows Razor components to be rendered into an embedded Web View, allowing them to run natively on mobile and desktop devices. In this rendering model, Razor components load and execute very quickly and have full access to the native capabilities of the device through the .NET platform. We will also explain how you can get the most out of your code assets by leveraging the exact same web UI components across mobile, desktop, and web. 

This is the 3rd year in a row where Oqtane is represented at a .NET Conf launch event :)

**Comment by engsky** [comment]
Also can you take us to the future of MAUI about performance since WPF styles make some load in memory .. Thank you. we love you Mr. @sbwalker

**Comment by sbwalker** [comment]
Here is the direct link to the session tonight: https://youtu.be/xyus7zlmgdc
